### ISSUE TITLE
Issues at trying to install on 9.x version.

### ISSUE THREAD

**Issue by Messhias** [TheDoctor0/laravel-mailjet-driver#6]
I'm trying to add the package in a new fresh laravel installation in version 9.x and I'm getting this one:

```
Problem 1
    - thedoctor0/laravel-mailjet-driver[1.0.0, ..., 1.0.1] require laravel/framework ~5.1|^6.0|^7.0 -> found laravel/framework[v5.1.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev] but it conflicts with your root composer.json require (^9.0).
    - thedoctor0/laravel-mailjet-driver[1.0.2, ..., 1.0.3] require laravel/framework ~5.1|^6.0|^7.0|^8.0 -> found laravel/framework[v5.1.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but it conflicts with your root composer.json require (^9.0).
```

Thank you,

**Comment by Messhias** [comment]
@TheDoctor0 any change to take look at this?

**Comment by TheDoctor0** [comment]
Hi @Messhias, sorry that you had to wait.
I just created a fresh Laravel 9 project and have no trouble installing the package.
Looking at the error it seems to be only fetching 1.0.0-1.0.3 versions but the Laravel 9 support has been added in [1.0.4](https://github.com/TheDoctor0/laravel-mailjet-driver/releases/tag/1.0.4).
I would advise explicitly executing `composer require thedoctor0/laravel-mailjet-driver:1.0.4`.
